distros/sample/gradle.properties

```diff
@@ -0,0 +1,4 @@
+# The Sample distro is not really a complete set of modules, but can be used as a minimal base developer workspace
+# It will cause several additional modules to be fetched as binaries, but consists of overall less source
+extraModules=CoreSampleGameplay,Sample
```

Comment:
`CoreSampleGameplay` itself depends on other things - should these modules be listed here?

Reply:
Yes and no - we're in a weird case here which might need a bit of reworking the logic. It again works as a good target for `init` (an example of something where you'd get jars fetched via Gradle when preparing your workspace) yet makes a poor definition for a zip, where yeah the lack of dependencies in the list here would lead to a broken zip. Odds are it would be better to either split apart the distro lists for `init` vs the zip packages _or_ when using for zips then resolve all the dependencies and shove them into the zip as well